timesketch/frontend-ng/src/views/Canvas.vue

```diff
@@ -0,0 +1,43 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+    <div v-show="currentRouteName === 'Explore'">
+      <explore :sketch-id="sketchId"></explore>
+    </div>
+    <div v-show="currentRouteName === 'Intelligence'">
+      <threat-intel :sketch-id="sketchId"></threat-intel>
+    </div>
+  </div>
+</template>
```

Comment:
As discussed offline, can we add a comment here about what this canvas view is supposed to accomplish?

Reply:
Done